tfrecorder/accessor.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, pandas_obj):
   def to_tfr(
       self,
       output_dir: str,
-      schema: input_schema.Schema = input_schema.ImageCsvSchema,
+      schema: input_schema.Schema = input_schema.IMAGECSVSCHEMA,
```

Comment:
Suggest using IMAGE_CSV_SCHEMA for readability.

Reply:
That's a lot cleaner, you're right.  Fixed.